client/a8c-for-agencies/components/form/field/index.tsx

```diff
@@ -7,20 +7,27 @@ type Props = {
 	label: string;
 	sub?: string;
 	description?: string;
-	isOptional?: boolean;
+	showOptionalLabel?: boolean;
 	children: ReactNode;
+	isRequired?: boolean;
 };
 
-export default function FormField( { label, sub, children, description, isOptional }: Props ) {
+export default function FormField( {
+	label,
+	sub,
+	children,
+	description,
+	showOptionalLabel,
+	isRequired,
+}: Props ) {
 	const translate = useTranslate();
 
 	return (
 		<div className="a4a-form__section-field">
 			<div className="a4a-form__section-field-heading">
 				<h3 className="a4a-form__section-field-label">
-					{ label }
-
-					{ isOptional && (
+					{ label } { isRequired && <span className="a4a-form__section-field-required">*</span> }
+					{ showOptionalLabel && (
```

Comment:
`! isRequired`

We only want to display the `optional` label if the field is not required, `isRequired = false`

Reply:
`! isRequired && showOptionalLabel && ...`